packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -764,6 +764,77 @@ void main() {
     expect(item5material.color, Colors.transparent); // the previous item should not be highlighted.
   }, variant: TargetPlatformVariant.desktop());
 
+  // Regression test for https://github.com/flutter/flutter/issues/147253.
+    testWidgets('Down key and up key can navigate on desktop platforms when a label text contains another label text',
```

Comment:
Too much indentation on this line (768).
Consider formatting these test declaration (lines 768 and 769) similarly to the following test (lines 838-839).